docs/messaging/azure-messaging.md

```diff
@@ -12,15 +12,6 @@ The Semantic Conventions for [Azure Service Bus](https://learn.microsoft.com/azu
 
 `messaging.system` MUST be set to `"servicebus"`.
 
-### Span names
-
-The span name SHOULD follow [the general messaging span name pattern](../messaging/azure-messaging.md): it SHOULD start with the messaging destination name (Event Hubs queue or topic name) and contain a low-cardinality name of the operation the span describes:
-
-- Spans names for `settle` operations SHOULD follow the `<destination name> {messaging.servicebus.disposition_status}` pattern.
```

Comment:
I think some parts are lost in this PR, but I'll send a follow-up to add info on operation names for Azure messaging systems along with examples and some other minor improvements, so it's ok.